nautobot/core/templates/inc/nav_favorites.html

```diff
@@ -0,0 +1,31 @@
+<ul class="list-unstyled">
+    <li class="sidenav-link-group text-center{% if request.user.navbar_favorites %} d-none{% endif %}"
+        style="text-transform: none;" id="favorites-empty">
+        You don't have any favorite pages yet.
+    </li>
+
+    {% for item in request.user.navbar_favorites %}
+        {% if item.link and item.name %}
```

Comment:
What would be the case where this isn't true? Should we have an `else` that renders a warning or error or anything?